config/datastores/formatters.yaml

```diff
@@ -1,5 +1,11 @@
 # This file gives the mapping between DatasetType and the
 # `lsst.daf.butler.Formatter` that handles it.
+writeRecipes: !include writeRecipes.yaml
+default:
+  lsst.obs.base.fitsExposureFormatter.FitsExposureFormatter:
+    # default is the default recipe regardless but this demonstrates
+    # how to specify a default write parameter
+    recipe: default
```

Comment:
I imagine this is the best place for it now, but I'd be surprised if we didn't eventually want a way to have obs_base provide some default butler configuration.